main.nf

```diff
@@ -2983,40 +3680,67 @@ compressVCFOutVEP = compressVCFOutVEP.dump(tag:'VCF')
 // STEP MULTIQC
 
 process MultiQC {
-    publishDir "${params.outdir}/Reports/MultiQC", mode: params.publishDirMode
+    publishDir "${params.outdir}/Reports/MultiQC", mode: params.publish_dir_mode
 
     input:
-        file (multiqcConfig) from Channel.value(params.multiqc_config ? file(params.multiqc_config) : "")
-        file (versions) from yamlSoftwareVersion
+        file (multiqcConfig) from ch_multiqc_config
+        file (mqc_custom_config) from ch_multiqc_custom_config.collect().ifEmpty([])
+        file (versions) from ch_software_versions_yaml.collect()
+        file workflow_summary from ch_workflow_summary.collectFile(name: "workflow_summary_mqc.yaml")
         file ('bamQC/*') from bamQCReport.collect().ifEmpty([])
         file ('BCFToolsStats/*') from bcftoolsReport.collect().ifEmpty([])
         file ('FastQC/*') from fastQCReport.collect().ifEmpty([])
-        file ('MarkDuplicates/*') from markDuplicatesReport.collect().ifEmpty([])
+        file ('TrimmedFastQC/*') from trimGaloreReport.collect().ifEmpty([])
+        file ('MarkDuplicates/*') from duplicates_marked_report.collect().ifEmpty([])
+        file ('DuplicatesMarked/*.recal.table') from baseRecalibratorReport.collect().ifEmpty([])
         file ('SamToolsStats/*') from samtoolsStatsReport.collect().ifEmpty([])
         file ('snpEff/*') from snpeffReport.collect().ifEmpty([])
         file ('VCFTools/*') from vcftoolsReport.collect().ifEmpty([])
 
     output:
-        set file("*multiqc_report.html"), file("*multiqc_data") into multiQCOut
+        file "*multiqc_report.html" into ch_multiqc_report
+        file "*_data"
+        file "multiqc_plots"
 
     when: !('multiqc' in skipQC)
 
     script:
+    rtitle = custom_runName ? "--title \"$custom_runName\"" : ''
+    rfilename = custom_runName ? "--filename " + custom_runName.replaceAll('\\W','_').replaceAll('_+','_') + "_multiqc_report" : ''
+    custom_config_file = params.multiqc_config ? "--config $mqc_custom_config" : ''
     """
-    multiqc -f -v .
+    multiqc -f ${rtitle} ${rfilename} ${custom_config_file} .
     """
 }
 
-multiQCOut.dump(tag:'MultiQC')
+ch_multiqc_report.dump(tag:'MultiQC')
 
-/*
- * Completion e-mail notification
- */
+// Output Description HTML
+process Output_documentation {
+    publishDir "${params.outdir}/pipeline_info", mode: params.publish_dir_mode
+
+    input:
+        file output_docs from ch_output_docs
+
+    output:
+        file "results_description.html"
```

Comment:
Do you use any of the logos under docs/images in the docs themselves? If so might want to also `input` them:

e.g.
https://github.com/nf-core/eager/blob/bf9256a8595e7c32e4e1721fa5ad8ca9997bed2d/main.nf#L479
https://github.com/nf-core/eager/blob/bf9256a8595e7c32e4e1721fa5ad8ca9997bed2d/main.nf#L2500

Reply:
Good point, but I don't think I have any pictures in the output docs at the moment

Reply:
Forget my comment further up then too ;-)